From 17bf7c681eaf5bb97c00e69921b64903d44f72e5 Mon Sep 17 00:00:00 2001
From: Peter Qiu <zqiu@chromium.org>
Date: Tue, 2 Jun 2015 14:03:54 -0700
Subject: [PATCH] Add option definition for Web Proxy Auto-Discovery

This option can be use to request Web Proxy Auto-Discovery info
from the DHCP server.

BUG=chromium:253915
TEST=Packet capture from network_DhcpNegotiationSuccess autotest
Turn on dhcp debugging in shill and note that a WebProxyAutoDiscoveryUrl
key is received (but ignored).

Author: Paul Stewart <pstew@chromium.org>
Reviewed-on: https://gerrit.chromium.org/gerrit/59967
---
 dhcpcd-definitions.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dhcpcd-definitions.conf b/dhcpcd-definitions.conf
index 58eba07..0e5aa3b 100644
--- a/dhcpcd-definitions.conf
+++ b/dhcpcd-definitions.conf
@@ -282,6 +282,9 @@ encap 0		string			nvt
 encap 1		binhex			vpn_id
 encap 255	flag			global
 
+# Web Proxy Auto-Discovery
+define 252  string          wpad_url
+
 # Options 222 and 223 are unused, RFC3942
 # Options 224-254 are reserved for Private Use
 # Option 255 End
-- 
2.2.0.rc0.207.ga3a616c

